File: backend/src/main/java/org/example/backend/domain/postcomment/dto/MyPostCommentReadResponseDto.java
package org.example.backend.domain.postcomment.dto;

import org.example.backend.domain.post.entity.Post.BoardType;
import org.example.backend.domain.post.entity.Post.Category;

public record MyPostCommentReadResponseDto(
    Long id,
    Long postId,
    String postTitle,
    String content,
    BoardType boardType,
    Category category
) {}

File: backend/src/main/java/org/example/backend/domain/postcomment/dto/MyPostCommentReadResponseDto.kt
package org.example.backend.domain.postcomment.dto

import org.example.backend.domain.post.entity.Post

@JvmRecord
data class MyPostCommentReadResponseDto(
    val id: Long,
    val postId: Long,
    val postTitle: String,
    val content: String,
    val boardType: Post.BoardType,
    val category: Post.Category?
)
